src/components/Header/Notifications/Invite.js

```diff
@@ -0,0 +1,41 @@
+import './invite.css';
+import React, {useRef} from 'react';
+
+const Invite=(props) =>{
+
+    const usernameRef = useRef(null);
+    const roleRef = useRef(null);
+
+    
+
+    const sendInvite=async()=>{
+        const val = usernameRef.current.value;
+    const valBoard = props.id;
+    const valRole = roleRef.current.value;
+         const res = await fetch('http://localhost:9090/users/invitations/'+val+'/boards/'+valBoard+'/roles/'+valRole+'', {
+                method: 'POST',
+                headers: {
+                    'Content-type': 'application/json',
+                    'Authorization': 'Bearer ' + sessionStorage.getItem('jwtToken')
+                },
+            },{});
+        props.close(false);
+    }
```

Comment:
Looks good
Just try to click "Ctrl + Alt + L" - it will help you to fix code formatting